### PR TITLE
Clean up some minor issues in the codebase

### PR DESCRIPTION
**Pull request by pabs3**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ instructions for doing that is available here:
 https://help.github.com/articles/creating-a-pull-request
 
 If you have found a bug, but do not have an accompanying patch to fix it, you
-can submit an issue report [via the web](http://rt.cpan.org/Public/Dist/Display.html?Name=File::LibMagic)
+can submit an issue report [via the web](https://rt.cpan.org/Public/Dist/Display.html?Name=File-LibMagic)
```
Comment (autarch) [inline CONTRIBUTING.md]: This file is autogenerated and I've already fixed this in the source template.

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ on Red Hat run:
 
     sudo yum install file-devel
 
-On Mac you can use homebrew (http://brew.sh/):
+On Mac you can use homebrew (https://brew.sh/):
```
Comment (autarch) [inline README.md]: This file is autogenerated from the `File::LibMagic` pod.

**Comment by autarch** [comment]
Merged from the CLI. Thanks!

**Comment by pabs3** [comment]
Thanks for merging some of the changes.

I wasn't able to find out how to regenerate CONTRIBUTING.md and
README.md, how can I get the rest of the changes merged?

These are the remaining changes:

https://github.com/pabs3/File-LibMagic/commit/2e083c75a2658c10a3cb96d1c13bf0b95101c66d

-- 
bye,
pabs

https://wiki.debian.org/PaulWise

**Comment by autarch** [comment]
Those are automatically updated when I do a release and committed as part of that process. I generally prefer not to update them in between releases since it muddies the commit history a bit.

**Comment by pabs3** [comment]
OK, that makes sense, thanks for the info.

-- 
bye,
pabs

https://wiki.debian.org/PaulWise

**Pull request closed without merging**